### PR TITLE
[update] ALTER MATERIALIZED VIEW (Continuous Aggregate)

### PR DESCRIPTION
**Pull request by Rajakavitha1**
# Description

Add the clause to rename a column in a continuous aggregate

# Links

Fixes https://github.com/timescale/docs/issues/2482

# Writing help

For information about style and word usage, see the [style guide](https://docs.timescale.com/about/latest/contribute-to-docs/)

# Review checklists

Reviewers: use this section to ensure you have checked everything before approving this PR:

## Subject matter expert (SME) review checklist

*   [ ] Is the content technically accurate?
*   [ ] Is the content complete?
*   [ ] Is the content presented in a logical order?
*   [ ] Does the content use appropriate names for features and products?
*   [ ] Does the content provide relevant links to further information?

## Documentation team review checklist

*   [ ] Is the content free from typos?
*   [ ] Does the content use plain English?
*   [ ] Does the content contain clear sections for concepts, tasks, and references?
*   [ ] Have any images been uploaded to the correct location, and are resolvable?
*   [ ] If the page index was updated, are redirects required
      and have they been implemented?
*   [ ] Have you checked the built version of this content?

### CODE REVIEW

File: api/alter_materialized_view.md [hunk 1]
```diff
@@ -11,12 +11,14 @@ api:
...
 
-`ALTER MATERIALIZED VIEW` statement can be used to modify some of the `WITH` clause [options][create_materialized_view] for the continuous aggregate view.
+`ALTER MATERIALIZED VIEW` statement can be used to modify some of the `WITH`
+clause [options][create_materialized_view] for the continuous aggregate view.
 `ALTER MATERIALIZED VIEW` statement also supports the following
 [PostgreSQL clauses][postgres-alterview] on the
 continuous aggregate view:
 
 *   `RENAME TO` clause to rename the continuous aggregate view;
+*   `RENAME [ COLUMN ]` clause to rename the continuous aggregate column;
```
Comment (Loquacity) [inline api/alter_materialized_view.md:21]:
```suggestion
*   `RENAME [COLUMN]` clause to rename the continuous aggregate column
```

File: api/alter_materialized_view.md [hunk 2]
```diff
@@ -11,12 +11,14 @@ api:
...
 # ALTER MATERIALIZED VIEW (Continuous Aggregate) <Tag type="community">Community</Tag>
 
-`ALTER MATERIALIZED VIEW` statement can be used to modify some of the `WITH` clause [options][create_materialized_view] for the continuous aggregate view.
+`ALTER MATERIALIZED VIEW` statement can be used to modify some of the `WITH`
+clause [options][create_materialized_view] for the continuous aggregate view.
 `ALTER MATERIALIZED VIEW` statement also supports the following
 [PostgreSQL clauses][postgres-alterview] on the
 continuous aggregate view:
 
 *   `RENAME TO` clause to rename the continuous aggregate view;
```
Comment (Loquacity) [inline api/alter_materialized_view.md:20]:
```suggestion
*   `RENAME TO` clause to rename the continuous aggregate view
```

File: api/alter_materialized_view.md [hunk 3]
```diff
@@ -11,12 +11,14 @@ api:
...
+*   `RENAME [ COLUMN ]` clause to rename the continuous aggregate column;
 *   `SET SCHEMA` clause to set the new schema for the continuous aggregate view;
```
Comment (Loquacity) [inline api/alter_materialized_view.md:22]:
```suggestion
*   `SET SCHEMA` clause to set the new schema for the continuous aggregate view
```

File: api/alter_materialized_view.md [hunk 4]
```diff
@@ -11,12 +11,14 @@ api:
...
 *   `SET TABLESPACE` clause to move the materialization of the continuous
   aggregate view to the new tablespace;
```
Comment (Loquacity) [inline api/alter_materialized_view.md:24]:
```suggestion
  aggregate view to the new tablespace
```

Comment (Loquacity) [reply api/alter_materialized_view.md:24]: And take the period off L25, too 👇

**Review by Loquacity — COMMENTED** [review]
Also need to compress the whitespace in the table L33-35

**Review by Loquacity — APPROVED** [review]

**Review by Loquacity — APPROVED** [review]

**Review by fabriziomello — APPROVED** [review]

**Pull request merged**